saleor/settings.py

```diff
@@ -215,15 +217,9 @@
     }
 }
 
-AUTHENTICATION_BACKENDS = (
-    'saleor.registration.backends.EmailPasswordBackend',
-    'saleor.registration.backends.ExternalLoginBackend',
-    'saleor.registration.backends.TrivialBackend'
-)
-
 AUTH_USER_MODEL = 'userprofile.User'
 
-LOGIN_URL = '/account/login'
+LOGIN_URL = '/accounts/login'
```

Comment:
Why is it `accounts`? From the user's perspective there is only one account they're logged into.